test/functional/test-viewport.js

```diff
@@ -1783,7 +1785,8 @@ describe('createViewport', () => {
     it('should bind to "iOS embed" when iframed', () => {
       win.parent = {};
       const ampDoc = installDocService(win, true).getAmpDoc();
-      const viewer = installViewerServiceForDoc(ampDoc);
+      installViewerServiceForDoc(ampDoc);
```

Comment:
Every test does this so it can be moved to `beforeEach`.

Ditto for the ampdoc service, though you may want to make that change in the other PR.